terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -917,6 +954,32 @@ then
       validation_return_code=2
     fi
   fi
+  if [ "$secure_mode" = "true" ]; then
+    res=$(check_tcp_port_open_in_seclist_or_nsg $MANAGED_SRV_NSG_OCID ${MS_ADMIN_PORT} "$wls_subnet_cidr_block" "nsg")
+    if [[ $res == *"WARNING"* ]]
+    then
+      for warning in "${res[@]}"; do
+        echo "$warning"
+      done
+    elif [[ $res -ne 0 ]]
+    then
+      echo "ERROR: Port ${MS_ADMIN_PORT} is not open for access by WLS Subnet CIDR [$wls_subnet_cidr_block] in Managed Server NSG [$MANAGED_SRV_NSG_OCID]. ${NETWORK_VALIDATION_MSG}"
+      validation_return_code=2
+    fi
+  fi
+  if [ "$secure_mode" = "true" ]; then
+    res=$(check_tcp_port_open_in_seclist_or_nsg $MANAGED_SRV_NSG_OCID ${WLS_NM_PORT} "$wls_subnet_cidr_block" "nsg")
```

Comment:
Why is a check for node manager port access only for secured production mode? Wouldn't this be needed for all WebLogic installations?